Lib/test/test_dis.py

```diff
@@ -1448,224 +1449,263 @@ def jumpy():
-  Instruction(opname='POP_TOP', opcode=1, arg=None, argval=None, argrepr='', offset=418, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=0, argval=0, argrepr='', offset=420, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='COPY', opcode=120, arg=3, argval=3, argrepr='', offset=422, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='POP_EXCEPT', opcode=89, arg=None, argval=None, argrepr='', offset=424, starts_line=None, is_jump_target=False, positions=None),
-  Instruction(opname='RERAISE', opcode=119, arg=1, argval=1, argrepr='', offset=426, starts_line=None, is_jump_target=False, positions=None),
-]
+# either the fodder is modified or the bytecode generation changes. Delete
+# everything in the 'Generated Test Cases' block below and replace with the
+# output of _regenerate_expected (after removing the print() output from
+# outer(), which will appear at the beginning).
+
+def _format(op):
+    arg = "_" if op.arg is None else op.arg
+    line = "_" if op.starts_line is None else op.starts_line
+    if isinstance(op.argval, types.CodeType):
+        argval = f"code_object_{op.argval.co_name}"
+        argrepr = f"repr({argval})"
+    elif isinstance(op.argval, str):
+        argval = repr(op.argval)
+        argrepr = repr(op.argrepr)
+    else:
+        argval = op.argval
+        argrepr = repr(op.argrepr)
+    return (f"    ({op.opname!r:<28}, {op.opcode:>3}, {arg:>3}, {argval!s:>20}, "
+            f"{argrepr:>24}, {op.offset:>3}, {line:>4}, {op.is_jump_target!r}),")
+
+
+def _print_expected(code, name, first_line):
+    print(f"expected_opinfo_{name} = [inst(*args) for args in [")
+    print(f"    # {'opname':<24}, opcode, arg, {'argval':>20}, argrepr, offset, starts_line, is_jump_target")
+    for op in dis.get_instructions(code, first_line=first_line):
+        print(_format(op))
+    print("]]\n")
+
+
+def _regenerate_expected():
+    for code, name, first_line in [
+            (outer, "outer", expected_outer_line),
+            (outer(), "f", expected_f_line),
+            (outer()(), "inner", expected_inner_line),
+            (jumpy, "jumpy", expected_jumpy_line)
+    ]:
+        _print_expected(code, name, first_line)
+
+# _regenerate_expected()
+# End test case regeneration
+
+def inst(opname, opcode, arg, argval, argrepr, offset, starts_line, is_jump_target):
+    return dis.Instruction(opname=opname, opcode=opcode, arg=arg, argval=argval,
+                           argrepr=argrepr, offset=offset,
+                           starts_line=starts_line, is_jump_target=is_jump_target,
+                           positions=None)
+
+_ = None
+
+
+# Generated test cases
+
+# fmt: off
+
+expected_opinfo_outer = [inst(*args) for args in [
+    # opname                  , opcode, arg,               argval, argrepr, offset, starts_line, is_jump_target
```

Comment:
This row is not aligned to match the subsequent rows.
Same applies to the other expected output as well.

Reply:
that was the best i could do with the last three columns being short but having long names. would it be okay to lengthen the lines even more to widen those columns to the length of the column heading? i wasn't sure about that.

Reply:
I have made the requested changes; please review again.